<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUIClass</class>
 <widget class="QMainWindow" name="MainWindowUIClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>1004</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>非织造纤网三维重建程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>600</width>
      <height>800</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Movie_tab">
     <attribute name="title">
      <string>视频</string>
     </attribute>
     <widget class="CImageShow" name="image_show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Pic_tab">
     <attribute name="title">
      <string>图像</string>
     </attribute>
     <widget class="CImageShow" name="dst_image_show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>707</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="4" colspan="2">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>输入扫描位置：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" rowspan="2">
      <widget class="QLineEdit" name="x_position_scan">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" rowspan="2">
      <widget class="QLineEdit" name="y_position_scan">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>平台当前坐标位置：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="x_position_show">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLineEdit" name="y_position_show">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>z:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLineEdit" name="z_position_show">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="4" colspan="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>68</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>607</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="1">
        <widget class="QLineEdit" name="input_short_step">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>灯光调节：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>频域聚焦</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_focus">
         <property name="text">
          <string>自动聚焦</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QSlider" name="light_change">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>移动步长：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>小区域自动聚焦</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>图像融合步长：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="input_long_step">
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>融合图像数量：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="input_short_num">
         <property name="text">
          <string>150</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>840</y>
      <width>458</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_save">
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_scan_deep">
       <property name="text">
        <string>图像融合</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btn_scan">
       <property name="text">
        <string>地毯式扫描采集图像</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="btn_stop_scan">
       <property name="text">
        <string>停止扫描</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_up">
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_front">
       <property name="text">
        <string>front</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_left">
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_right">
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_back">
       <property name="text">
        <string>back</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btn_down">
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1462</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CImageShow</class>
   <extends>QOpenGLWidget</extends>
   <header>cimageshow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainWindowUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_reset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_up</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_front</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Front()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_right</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Right()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>-65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_left</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Left()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>-70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_back</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>-87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_down</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>-83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>light_change</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Light_Change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>-85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Save_Pic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1094</x>
     <y>945</y>
    </hint>
    <hint type="destinationlabel">
     <x>773</x>
     <y>-36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_focus</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Focus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>-26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_ROI_Focus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>-51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Fourer_Focus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>-108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_scan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Scan_Sample()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1180</x>
     <y>957</y>
    </hint>
    <hint type="destinationlabel">
     <x>1187</x>
     <y>-42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_scan_deep</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Scan_Deep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>957</y>
    </hint>
    <hint type="destinationlabel">
     <x>947</x>
     <y>-50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_stop_scan</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowUIClass</receiver>
   <slot>On_Stop_Scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1059</x>
     <y>1012</y>
    </hint>
    <hint type="destinationlabel">
     <x>1057</x>
     <y>-25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>On_Reset()</slot>
  <slot>On_Light_Change(int)</slot>
  <slot>On_Up()</slot>
  <slot>On_Down()</slot>
  <slot>On_Left()</slot>
  <slot>On_Right()</slot>
  <slot>On_Front()</slot>
  <slot>On_Back()</slot>
  <slot>On_Save_Pic()</slot>
  <slot>On_Focus()</slot>
  <slot>On_ROI_Focus()</slot>
  <slot>On_Fourer_Focus()</slot>
  <slot>On_Stop_Acquisition()</slot>
  <slot>On_Scan_Sample()</slot>
  <slot>On_Scan_Deep()</slot>
  <slot>On_Stop_Scan()</slot>
 </slots>
</ui>
